index.html

```diff
@@ -165,5 +165,21 @@ <h4 class="end-text">LETS'S CONNECT</h4>
             <hr>
         </article>
     </main>
+    <footer class='container'>
+        <article >
+            <h1>Contact me</h1>
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+            <form action="https://formspree.io/f/xayvwlyd" method="post" class="contacts-form">
+                <label for="username"></label>
+                <input id="username" type="text" placeholder="Username" required name="username">
+                <label for="email"></label>
+                <input id="email" type="email" placeholder="E-mail" required name="email">
+                <label for="text-form"></label>
+                <textarea name="text-form" id="text-form" cols="50" rows="6" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Please include the `maxlength=500` attribute for this section.

Reply:
I put it but it gives me a linter error so ,  i do it and ignore the errors ?